README.md

```diff
@@ -0,0 +1,4 @@
+# dFC
```

Comment:
more details needed. For example, you can explain each method, or what steps should be done to run your own data etc.